### PR TITLE
[platform]: update mitac platform drivers and device

### PR DESCRIPTION
**Pull request by stanepc1**
**- What I did**
Update ly1200 to support stretch
**- How I did it**
Modify driver code
**- How to verify it**
Upload image to DUT to ensure all modules are running correctly.
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_polld.c [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+ * FEATURE:
+ *   bms_xcvr_polld
+ * PURPOSE:
+ *   poll present pin for each port
+ * PARAMETERS:
+ *   args               (IN) NULL
+ * RETURNS:
+ *   No.
+ */
+void* bms_xcvr_polld(void *args)
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_polld.c]: this is essentially the xrcvd does.

Comment (stanepc1) [reply platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_polld.c]: If our xcvr-servd is not running in SONiC, we will not see "state UP" via the "ip addr" command and will not see anything through the "show interfaces transceiver eeprom" command when plugging in the transceiver.

The implementation difference between our xcvr-servd and SONiC's xrcvd might be related to our hardware design. Please allow our xcvr-servd to run on SONiC for our platform module.

File: platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_access.c [hunk 2]
```diff
@@ -0,0 +1,345 @@
...
+ *   bus                (IN) I2C bus to be looking for
+ *   addr               (IN) I2C slave address of the transceiver
+ *   offset             (IN) register offset
+ *   val                (IN) new register value
+ * RETURNS:
+ *   write success, this function returns 0.
+ *   for other cases, negative errno is returned.
+ */
+static int
+bms_xcvr_i2c_write(unsigned char bus,
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_access.c]: we have sff_8436_eeprom driver allow you user  to access sfp eeprom data via sysfs. It is redudant to have same code in user space that does the same thing.

File: platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_access.c [hunk 3]
```diff
@@ -0,0 +1,345 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_access.c]: the function here is overalpping with the sff_8436_eeprom driver in sonic.

Comment (stanepc1) [reply platform/broadcom/sonic-platform-modules-mitac/ly1200-32x/opt/xcvr-serv/bms_xcvr_access.c]: There are two main benefits to access EEPROM via the API of bms_xcvr_access.c
1) it is a portable implementation that does not depend on any EEPROM driver.
2) the execution performance is faster than calling three APIs open(), read()/write(), and close() to access sysfs.

**Comment by stanepc1** [comment]
remove minigraph.xml

**Comment by stanepc1** [comment]
We will modify the default level of low-speed pins for plug&play of QSFP and remove the controversial "xcvr_servd" of MiTAC.

**Pull request closed without merging**